query/src/main/kotlin/com/linecorp/kotlinjdsl/querydsl/where/WhereDsl.kt

```diff
@@ -5,5 +5,11 @@ import com.linecorp.kotlinjdsl.querydsl.expression.ExpressionDsl
 import com.linecorp.kotlinjdsl.querydsl.predicate.PredicateDsl
 
 interface WhereDsl : ExpressionDsl, PredicateDsl {
-    fun where(predicate: PredicateSpec)
+    fun where(predicate: PredicateSpec?)
+
+    fun whereAnd(vararg predicates: PredicateSpec?) = whereAnd(predicates.toList())
+    fun whereAnd(predicates: List<PredicateSpec?>)
+
+    fun whereOr(vararg predicates: PredicateSpec?) = whereOr(predicates.toList())
+    fun whereOr(predicates: List<PredicateSpec?>)
```

Comment:
-- korean
@jaeykweon 넵 말씀해주신 다른 인터페이스이기 때문입니다.

내부 구현이 동일하다는 것은 개발자만이 알고 있는 사실입니다. 개인적으로는 테스트는 내부 구현을 알고 작성한다기 보다 외부의 시점에서 작성하는 것이 맞다고 생각하기 때문에 vararg와 list 테스트를 별도로 가져가도록 말씀드렸습니다.

-- english
Yes, it's because different interface.

Only developers know that internal implementations are same. Personally, The vararg and list test should be separated  because I think it is right to write the test from an external perspective rather than knowing the internal implementation.